aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  Main,
+  Text,
+  TextInput,
+} from 'grommet';
+import { Close, Next } from 'grommet-icons';
+
+import { FormContainer } from '.';
+import { emailValidation } from './formHelpers';
+
+const ResetPassword = ({ closeLayer, email }) => {
+  const [formValues, setFormValues] = React.useState({ resetEmail: email });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your password reset logic here
+    // Display success status
+    closeLayer();
+  };
+
+  return (
+    <>
+      <Box
+        direction="row"
+        justify="end"
+        pad={{ horizontal: 'small', top: 'small' }}
+      >
+        <Button
+          a11yTitle="Close reset password form"
+          icon={<Close />}
+          onClick={closeLayer}
+        />
+      </Box>
+      <Box
+        gap="medium"
+        margin={{ horizontal: 'xlarge', bottom: 'xlarge', top: 'large' }}
+        width="medium"
+      >
+        <Heading level={2} margin="none">
+          Reset Password
+        </Heading>
+        <Form
+          validate="blur"
+          value={formValues}
+          onChange={setFormValues}
+          onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+        >
+          <Text>A password reset email will be sent to </Text>
+          <FormField
+            label="Email"
+            name="resetEmail"
+            component={TextInput}
+            type="email"
+            placeholder="your.email@company.com"
+            validate={emailValidation}
+          />
+          <Button label="Send Password Reset" primary type="submit" />
+        </Form>
+      </Box>
+    </>
+  );
+};
+
+export const SignInExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+  const [showForgotPassword, setShowForgotPassword] = React.useState(false);
+
+  const onClose = () => {
+    setShowForgotPassword(false);
+  };
+
+  const onForgotPassword = () => {
+    setShowForgotPassword(true);
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
```

Comment:
Really liked how you added in this comment to show they need to use their own onSubmit for their form!

Reply:
Agree!